prerequisites/install-kyverno.sh

```diff
@@ -2,6 +2,40 @@
 
 set -o errexit -o nounset -o pipefail
 
-helm repo add kyverno https://kyverno.github.io/kyverno/
-helm repo update
-helm install kyverno kyverno/kyverno -n kyverno --create-namespace --set replicaCount=1
+# Default installation method
+INSTALL_METHOD="airgap"
+
+# Parse command line arguments
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    --helm)
+      INSTALL_METHOD="helm"
+      shift
+      ;;
+    --airgap)
+      INSTALL_METHOD="airgap"
+      shift
+      ;;
+    -h|--help)
+      echo "Usage: $0 [--helm|--airgap]"
+      echo "  --helm    Install Kyverno using Helm"
+      echo "  --airgap  Install Kyverno using kyverno.yaml from the repository (default)"
+      exit 0
+      ;;
+    *)
+      echo "Unknown option $1"
+      echo "Use --help for usage information"
+      exit 1
+      ;;
+  esac
+done
+
+if [[ "$INSTALL_METHOD" == "helm" ]]; then
+  echo "Installing Kyverno using Helm..."
+  helm repo add kyverno https://kyverno.github.io/kyverno/
+  helm repo update
+  helm install kyverno kyverno/kyverno -n kyverno --create-namespace --set replicaCount=1
+elif [[ "$INSTALL_METHOD" == "airgap" ]]; then
+  echo "Installing Kyverno using kyverno.yaml (airgap mode)..."
+  kubectl create -f kyverno.yaml
```

Comment:
Where is the original `kyverno.yaml` from? We can add a comment with the URL so we can check for updates.

Reply:
this is from https://github.com/kyverno/kyverno/releases/download/v1.15.1/install.yaml